scripts/dynamic-import.js

```diff
@@ -0,0 +1,12 @@
+/**
+ * Get and require all files fetched using require.context method.
```

Comment:
This method just loops over paths. How does it require files? The requiring is done elsewhere, no?

Reply:
yes: